composer.json

```diff
@@ -44,6 +44,7 @@
         "symfony/yaml": "~2.3",
         "symfony/security-acl": "~2.4",
         "sonata-project/intl-bundle": "~2.1",
+        "sllh/php-cs-fixer-styleci-bridge": "~1.0",
```

Comment:
StyleCI aims to give the best results possible, and that means we use our own php-cs-fixer 2.x fork.